main.js

```diff
@@ -170,108 +179,106 @@ const createEventElement = (eventData) => {
 
 }
 
-function renderHomePage() {
+const handleAddToCart = (input, price, ticketId, date) => {
+  const quantity = input;
+  if (parseInt(quantity)){
+      fetch('https://localhost:7203/api/Order/addOrder', {
+        method:"POST",
+        headers:{
+          "Content-Type":"application/json",
+        },
+        body:JSON.stringify({
+          ticketCategoryId:+ticketId,
+          customerName:"Alexe Alexandru",
+          orderedAt:date,
+          numberOfTickets:+quantity,
+          totalPrice:+price,
+
+
+        })
+      }).then((response)=>{
+        return response.json().then((data) => {
+          if(!response.ok){
+            console.log("Something went wrong!");
```

Comment:
You might want to return here empty list. If response is not ok, so you application doesn't break.